modules/sensor-requirements.adoc

```diff
@@ -7,6 +7,12 @@
 Sensor monitors your Kubernetes and {ocp} clusters.
 These services currently deploy in a single deployment, which handles interactions with the Kubernetes API and coordinates with Collector.
 
+[IMPORTANT]
+====
+* In order to ensure optimal functionality it is recommended to keep your secured clusters up to date and on the same version as Central. Out of date Sensor versions are supported for 6 months, but may not have the full functionality of an up to date version.
+* Enabling automated upgrades for your secured clusters is strongly recommended. Should you prefer manually upgrading your secured clusters refer to the xref:../upgrading/upgrade-roxctl.adoc#upgrade-secured-clusters[manual on upgrading secured clusters].
+====
```

Comment:
I thought for a while about this content in `sensor-requirements.adoc` and I find it may look a bit odd. The problem, in my opinion, is that the section `Prerequisites for installing Sensor` talks only about Sensor whereas we want the entire Secured Cluster to have version consistent with Central. Secured Cluster != Sensor because Secured Cluster includes more than just Sensor.

I cannot come up with a better text here that would take the reader from Sensor to Secured Cluster context without referring to the architecture page.

I suggest to remove this from here and rely on notes that you already put in `upgrade*` pages. Additionally, I suggest to add a note to that section <https://docs.openshift.com/acs/3.72/installing/install-ocp-operator.html#install-acs-operator_install-ocp-operator>, right after the text I highlighted on the screenshot

![image](https://user-images.githubusercontent.com/537715/202462750-6cf2c768-770b-4f43-8aee-9376d7b69498.png)

The text can say something like
```adoc
[IMPORTANT]
====
If you choose manual updates, it is recommended to update ACS Operator in all
secured clusters consistently with the ACS Operator in the central cluster. This way the
secured clusters will have the same version as central which ensures the optimal
compatibility.
====
```

Reply:
My point is that this relationship between Sensor and Secured Cluster may not be clear to a reader that doesn't know ACS architecture and https://docs.openshift.com/acs/installing/prerequisites.html is the first documentation page the reader ever seen for ACS.